### PR TITLE
Add our security mailing list to the README

### PR DESCRIPTION
**Pull request by duglin**

### CODE REVIEW

**Comment by duglin** [comment]
Given this is the only topic for this week's call, and all of the SDK versions of this PR have been approved, if I can get someone to LGTM this then we can approve this (administrative) PR off-line and cancel the call tomorrow. Anyone wanna double check it for me?

**Review by pierDipi — APPROVED** [review]
LGTM

**Review by jskeet — APPROVED** [review]

**Comment by duglin** [comment]
thanks guys, but we're gonna have a call today anyway :-)

**Pull request merged**
